spec/plainyearmonth.html

```diff
@@ -627,26 +627,17 @@ <h1>
         1. Set _other_ to ? ToTemporalYearMonth(_other_).
         1. Let _calendar_ be _yearMonth_.[[Calendar]].
         1. If ? CalendarEquals(_calendar_, _other_.[[Calendar]]) is *false*, throw a *RangeError* exception.
-        1. Set _options_ to ? GetOptionsObject(_options_).
-        1. Let _disallowedUnits_ be « *"week"*, *"day"*, *"hour"*, *"minute"*, *"second"*, *"millisecond"*, *"microsecond"*, *"nanosecond"* ».
-        1. Let _smallestUnit_ be ? ToSmallestTemporalUnit(_options_, _disallowedUnits_, *"month"*).
-        1. Let _largestUnit_ be ? ToLargestTemporalUnit(_options_, _disallowedUnits_, *"auto"*, *"year"*).
-        1. Perform ? ValidateTemporalUnitRange(_largestUnit_, _smallestUnit_).
-        1. Let _roundingMode_ be ? ToTemporalRoundingMode(_options_, *"trunc"*).
-        1. If _operation_ is ~since~, then
-          1. Set _roundingMode_ to ! NegateTemporalRoundingMode(_roundingMode_).
-        1. Let _roundingIncrement_ be ? ToTemporalRoundingIncrement(_options_, *undefined*, *false*).
```

Comment:
MaximumTemporalDurationRoundingIncrement missing?

Reply:
In the status quo, DifferenceTemporalPlainYearMonth does not call MaximumTemporalDurationRoundingIncrement but pass *undefined* to the second arg of ToTemporalRoundingIncrement. However, in this case, since _disallowedUnits_ is « *"week"*, *"day"*, *"hour"*, *"minute"*, *"second"*, *"millisecond"*, *"microsecond"*, *"nanosecond"* » _smallestUnit_ can only be one of  *"year"*, *"month"*, or *"week"* but no other unit. And MaximumTemporalDurationRoundingIncrement of any of  *"year"*, *"month"*, or *"week"*  is always *undefined*, therefore it is equivlent.